src/com/pentalog/pentastagiu/homework2/HomeworkApp.java

```diff
@@ -3,5 +3,35 @@
 public class HomeworkApp {
```

Comment:
ti-am sugerat niste schimbari pe care le-ai putea face pentru a imbunatati codul iar pasul urmator ar fi sa incerci sa faci scenariile de test dinamice(sa citesti de la consola input introdus de cel care testeaza si in functie de asta sa faci diferite actiuni/scenarii de test)